addons/utm/tests/test_utm.py

```diff
@@ -34,10 +34,11 @@ def test_name_generation(self):
 
```

Comment:
Doesn't need to be its own commit

Reply:
why? i think explaining why tests were modified is good thing. but i will remove if you insist

Reply:
Well the commits should make sense independently as much as possible. Here:

if you take only the commit that makes the functional change -> this test breaks
if you take only this test -> it doesn't really make sense to change this

If the reason for the change in tests isn't obvious you can just make the commit message longer in the original commit :)